### ISSUE TITLE
Installation Troubleshooting

### ISSUE THREAD

**Issue by buddh4** [buddh4/humhub-docs#15]
- Wrong timezone
- No Mails are sent
- Cronjobs not working
- File Permissions Errors
- Check requirements

**Comment by Felli** [comment]
Might end up getting to use the information from the [Con Job Setup](https://community.humhub.com/s/installation-and-setup/wiki/page/view?title=Cron+Job+Setup) wiki for Troubleshooting for Cronjobs not working.

**Comment by buddh4** [comment]
Hmm, perhaps we should add this as extra section within the admin guide, looks nice by the way! ;)

**Comment by Felli** [comment]
I agree, an extra section in the admin guide would be a good idea, also I'd like to credit both [Joel Klein](https://community.humhub.com/u/joel+klein/) & [Stefan Zilz](https://community.humhub.com/u/elchupete/) from the community with some help with Plesk & cPanel configurations. :+1:
